core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -90,6 +92,54 @@ package object config {
     .toSequence
     .createWithDefault(Nil)
 
+  // Blacklist confs
+  private[spark] val BLACKLIST_ENABLED =
+    ConfigBuilder("spark.blacklist.enabled")
+      .booleanConf
+      .createOptional
+
+  private[spark] val MAX_TASK_ATTEMPTS_PER_EXECUTOR =
+    ConfigBuilder("spark.blacklist.task.maxTaskAttemptsPerExecutor")
+      .intConf
+      .createWithDefault(1)
+
+  private[spark] val MAX_TASK_ATTEMPTS_PER_NODE =
+    ConfigBuilder("spark.blacklist.task.maxTaskAttemptsPerNode")
```

Comment:
The difference is that the task attempt is the same task failing multiple attempts vs maxFailedTasks is the number any failed tasks (ie different tasks) and I think it was just trying to differentiate the 2

I don't remember if we discussed actual naming in the design doc, I think he was just trying to differentiate the 2. We probably could rename to something like maxFailedTaskAttempts but that is pretty close to maxFailedTasks so not sure if that would be more confusing. @squito  can correct me if I'm wrong.